test/rules/static-this/test.ts.lint

```diff
@@ -0,0 +1,14 @@
+class StaticThis {
```

Comment:
I've updated the tests' set. I didn't append `static getSmth = () => this._value` case there because `tcs` throws an error on this: `'this' cannot be referenced in a static property initializer.`